packages/client/package.json

```diff
@@ -66,6 +67,7 @@
     "graphql-tools": "^4.0.7",
     "handlebars": "^4.7.6",
     "history": "^4.7.2",
+    "html-to-pdfmake": "^2.5.13",
```

Comment:
seems to be [a fairly lightweight](https://github.com/Aymkdn/html-to-pdfmake/blob/master/index.js) dependency, should be fine for bundle size👍